agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporter.java

```diff
@@ -211,7 +81,7 @@ public CompletableResultCode export(Collection<SpanData> spans) {
     }
 
     CompletableResultCode overallResult = new CompletableResultCode();
-    CompletableResultCode exportResult = exportFn.apply(telemetryItems);
+    CompletableResultCode exportResult = telemetryItemExporter.send(telemetryItems);
     boolean mappingFailureFinal = mappingFailure;
     exportResult.whenComplete(
```

Comment:
do we need to handle this here?

Reply:
I think it's more accurate, because in case of standalone exporter (unlike the agent exporter) a caller may want to know when the export finishes and if there were any errors